inference-engine/src/legacy_api/src/graph_transformer.cpp

```diff
@@ -221,6 +221,7 @@ static std::vector<std::string> skipConstInfer = {
     "Squeeze",
     "TensorIterator",
     "LSTMSequence",
+    "MVN"
```

Comment:
Coming back to original root cause - why ngraph does not fold this constant?

Reply:
I faced with some issues in the reference implementation. I propose to do it separately.

Reply:
Ok, please, create a ticket for this.